src/customHooks/useAuthorize.js

```diff
@@ -0,0 +1,17 @@
+import { useEffect } from 'react';
+
+const URL = 'http://localhost:5020/auth';
+
+const useAuthorize = () => {
+  useEffect(() => {
+    (async () => {
+      if (!localStorage.getItem('token')) {
```

Comment:
you have a string that you're using in different places in the app. It's very easy to do a typo and spend too much time wondering what's happening and what broke. Please move it to a constant you can use throughout the app.